cmd/gateway-conformance/main.go

```diff
@@ -129,20 +144,55 @@ func main() {
 						cmd.Env = append(cmd.Env, fmt.Sprintf("SUBDOMAIN_GATEWAY_URL=%s", subdomainGatewayURL))
 					}
 
-					cmd.Stdout = out{output}
+					cmd.Stdout = out{
+						Writer: output,
+						Filter: func(line string) bool {
+							return verbose || strings.Contains(line, "FAIL") || strings.HasPrefix(line, "PASS")
```

Comment:
Could you use `HasPrefix(line, "--- FAIL:")` here?

Reply:
I might miss something though, is this capturing the regular output we get in CI?

like:

```
--- PASS: TestTrustlessCarOrderAndDuplicates/GET_CAR_with_dups=n_of_UnixFS_Directory_With_Duplicate_Files/Body (0.00s)
=== NAME  TestTrustlessCarOrderAndDuplicates/GET_CAR_with_dups=n_of_UnixFS_Directory_With_Duplicate_Files
--- FAIL: TestTrustlessCarOrderAndDuplicates/GET_CAR_with_dups=n_of_UnixFS_Directory_With_Duplicate_Files (0.01s)
=== NAME  TestTrustlessCarOrderAndDuplicates
=== RUN   TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors
=== RUN   TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors/Header_Content-Type
--- PASS: TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors/Header_Content-Type (0.00s)
=== NAME  TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors
=== RUN   TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors/Header_Content-Type#01
    run.go:53: 
        Name: GET CAR with order=unk of UnixFS Directors
        Hint: 
        				The response with order=unk MUST contain all the blocks required to construct
        				the requested CID. However, it can be in any order and duplicates MAY occur.
        			
        
        Error: Header 'Content-Type' expect to find substring 'order=', got 'application/vnd.ipld.car; version=1'
        
        Request:
        {
          "method": "GET",
          "path": "/ipfs/bafybeihchr7vmgjaasntayyatmp5sv6xza57iy2h4xj7g46bpjij6yhrmy",
          "headers": {
            "Accept": "application/vnd.ipld.car; version=1; order=unk"
          }
        }
        
        Expected Response:
        {
          "statusCode": 200,
          "headers": [
            {
              "key": "Content-Type",
              "check": {}
            },
            {
              "key": "Content-Type",
              "check": {}
            }
          ],
          "body": {}
        }
        
        Actual Request:
        GET /ipfs/bafybeihchr7vmgjaasntayyatmp5sv6xza57iy2h4xj7g46bpjij6yhrmy HTTP/1.1
        Host: 127.0.0.1:8080
        User-Agent: Go-http-client/1.1
        Accept: application/vnd.ipld.car; version=1; order=unk
        Accept-Encoding: gzip
        
        
        
        Actual Response:
        HTTP/1.1 200 OK
        Transfer-Encoding: chunked
        Accept-Ranges: none
        Access-Control-Allow-Headers: Content-Type
        Access-Control-Allow-Headers: Range
        Access-Control-Allow-Headers: User-Agent
        Access-Control-Allow-Headers: X-Requested-With
        Access-Control-Allow-Methods: GET
        Access-Control-Allow-Origin: *
        Access-Control-Expose-Headers: Content-Length
        Access-Control-Expose-Headers: Content-Range
        Access-Control-Expose-Headers: X-Chunked-Output
        Access-Control-Expose-Headers: X-Ipfs-Path
        Access-Control-Expose-Headers: X-Ipfs-Roots
        Access-Control-Expose-Headers: X-Stream-Output
        Cache-Control: public, max-age=29030400, immutable
        Content-Disposition: attachment; filename="bafybeihchr7vmgjaasntayyatmp5sv6xza57iy2h4xj7g46bpjij6yhrmy.car"; filename*=UTF-8''bafybeihchr7vmgjaasntayyatmp5sv6xza57iy2h4xj7g46bpjij6yhrmy.car
        Content-Type: application/vnd.ipld.car; version=1
        Date: Fri, 23 Jun 2023 09:57:20 GMT
        Etag: W/"bafybeihchr7vmgjaasntayyatmp5sv6xza57iy2h4xj7g46bpjij6yhrmy.car.d8eotdcfofg54"
        X-Content-Type-Options: nosniff
        X-Ipfs-Path: /ipfs/bafybeihchr7vmgjaasntayyatmp5sv6xza57iy2h4xj7g46bpjij6yhrmy
        X-Ipfs-Roots: bafybeihchr7vmgjaasntayyatmp5sv6xza57iy2h4xj7g46bpjij6yhrmy
        
        
        
--- FAIL: TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors/Header_Content-Type#01 (0.00s)
=== NAME  TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors
=== RUN   TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors/Body
--- PASS: TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors/Body (0.00s)
=== NAME  TestTrustlessCarOrderAndDuplicates/GET_CAR_with_order=unk_of_UnixFS_Directors
```

Reply:
Updated, we have to watch out for U+0016